AdminApi.Sdk/Endpoints/Identities/Types/Requests/CreateIdentityRequest.cs

```diff
@@ -1,10 +1,13 @@
-﻿namespace Backbone.AdminApi.Sdk.Endpoints.Identities.Types.Requests;
+﻿using Backbone.AdminApi.Sdk.Endpoints.Challenges.Types;
+
+namespace Backbone.AdminApi.Sdk.Endpoints.Identities.Types.Requests;
 
 public class CreateIdentityRequest
 {
     public required string ClientId { get; set; }
     public required string ClientSecret { get; set; }
-    public required byte[] IdentityPublicKey { get; set; }
+    public required string IdentityPublicKey { get; set; }
```

Comment:
Why didn't you keep the byte[] type? Imo it's better, because in case of a string, you don't know that it has to be base64 encoded.